oaknational/Oak-Web-Application
# LESQ-271 feat(lessonOverview): add hasDownloadable onto 2023 query

## Description

add hasDownloadableResource field to query

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-1921--oak-web-application.netlify.thenational.academy/teachers-2023/programmes/combined-science-secondary-ks4-foundation-aqa/units/measuring-waves/lessons/representing-transverse-waves
2. Click on _______
3. You should see _______

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
